src/components/icon.cjsx

```diff
@@ -24,6 +24,7 @@ module.exports = React.createClass
   render: ->
     classNames = classnames('tutor-icon', 'fa', "fa-#{@props.type}", @props.className, {
       'fa-spin': @props.spin
+      'clickable': @props.tooltip and @props.tooltipProps.trigger is 'click'
```

Comment:
Hm, well it will add the `cursor: pointer` style, but I think that's a "good thing".  I attempted to find if there were spots in the styles where we were already setting it manually so they could be removed by using `grep -r -C5 cursor ./resources/styles*`, but didn't spot any.